test/e2e/node/pod_resize.go

```diff
@@ -436,6 +413,134 @@ func doPodResizeSchedulerTests(f *framework.Framework) {
 	})
 }
 
+func doPodResizeRetryDeferredTests(f *framework.Framework) {
+	ginkgo.It("pod-resize-retry-deferred-tests", func(ctx context.Context) {
+		podClient := e2epod.NewPodClient(f)
+		nodes, err := e2enode.GetReadySchedulableNodes(ctx, f.ClientSet)
+		framework.ExpectNoError(err, "failed to get running nodes")
+		gomega.Expect(nodes.Items).ShouldNot(gomega.BeEmpty())
+		framework.Logf("Found %d schedulable nodes", len(nodes.Items))
+
+		ginkgo.By("Find node CPU resources available for allocation!")
+		node := nodes.Items[0]
+		nodeAllocatableMilliCPU, nodeAvailableMilliCPU := getNodeAllocatableAndAvailableMilliCPUValues(ctx, f, &node)
+		framework.Logf("Node '%s': NodeAllocatable MilliCPUs = %dm. MilliCPUs currently available to allocate = %dm.",
+			node.Name, nodeAllocatableMilliCPU, nodeAvailableMilliCPU)
+
+		//
+		// 	   1. Create pod1 and pod2 and pod3 on node.
+		// 	   2. Resize pod3 to request more cpu than available, verify the resize is deferred.
+		//	   3. Resize pod1 down to make space for pod3, verify pod3's resize has completed.
```

Comment:
Maybe add another version of this test (or another step) that deletes pod1 to make room for pod3 instead?

Reply:
that test already exists: https://github.com/kubernetes/kubernetes/blob/7e1cec680e55244ccc31d404065ee12cefaa2578/test/e2e/node/pod_resize.go#L372